src/main/java/dev/tycho/stonks/managers/DatabaseHelper.java

```diff
@@ -857,6 +858,246 @@ public void changeVerification(Player player, String companyName, boolean newVer
+      player.sendMessage(ChatColor.RED + "SQL EXCEPTION TELL WHEEZY");
+      return;
+    }
+    if (service == null) {
+      player.sendMessage(ChatColor.RED + "Service id not found");
+      return;
+    }
+
+    Subscription subscription = service.getSubscription(player);
+    if (subscription == null) {
+      player.sendMessage(ChatColor.RED + "You are not subscribed to this service");
+      return;
+    }
+
+    if (!subscription.isOverdue()) {
+      player.sendMessage(ChatColor.RED + "The subscription is not overdue, you don't need to pay for it");
+      return;
+    }
+
+
+    new ConfirmationGui.Builder().title("Pay bill of $" + service.getCost()).onChoiceMade(
+        c -> {
+          if (!c) return;
+          if (!Stonks.economy.withdrawPlayer(player, service.getCost()).transactionSuccess()) {
+            player.sendMessage(ChatColor.RED + "Insufficient funds!");
+            return;
+          } else {
+            //Payment success
+            try {
+              //Update that the subscription is paid
+              subscription.registerPaid();
+              databaseManager.getSubscriptionDao().update(subscription);
+              service.getAccount().getAccount().addBalance(service.getCost());
+              databaseManager.logTransaction(new Transaction(service.getAccount(),
+                  player.getUniqueId(), "Subscription payment for " + service.getName(), service.getCost()));
+
+              //Update the account database
+              databaseManager.updateAccount(service.getAccount().getAccount());
+
+              //Subscription created!
+              player.sendMessage(ChatColor.GREEN + "You have resubscribed to the service " + service.getName() +
+                  " (provided by " + service.getCompany().getName() + ")!");
+              player.sendMessage(ChatColor.GREEN + "This service will expire in " + service.getDuration() + " days");
+              if (subscription.isAutoPay()) {
+                player.sendMessage(ChatColor.GREEN + "You have set your subscription to automatically renew, so you don't need to do anything.");
+              } else {
+                player.sendMessage(ChatColor.GREEN + "You have set your subscription to manually renew, so you will need to resubscribe in "
+                    + service.getDuration() + " days time.");
+              }
+            } catch (SQLException e) {
+              e.printStackTrace();
+              player.sendMessage(ChatColor.RED + "SQL EXCEPTION TELL WHEEZY");
+            }
+          }
+        }
+    ).open(player);
+
+  }
+
+  public void unsubscribeFromService(Player player, int id) {
+    Service service;
+    try {
+      service = databaseManager.getServiceDao().queryForId(id);
+    } catch (SQLException e) {
+      e.printStackTrace();
+      player.sendMessage(ChatColor.RED + "SQL EXCEPTION TELL WHEEZY");
+      return;
+    }
+    if (service == null) {
+      player.sendMessage(ChatColor.RED + "Service id not found");
```

Comment:
Use `sendMessage(Player, String)` and don't use `ChatColor.RED` (Also follow the messages i've used for errors)